src/chrome/create-chrome.ts

```diff
@@ -179,8 +179,9 @@ export const createChromeContext = ({
     clearAnsibleTrialFlag,
     isAnsibleTrialFlagActive,
     setAnsibleTrialFlag,
-    chromeHistory,
-    analytics: analytics!,
+    // FIXME: get rid of these anys
+    chromeHistory: chromeHistory as any,
```

Comment:
These should no longer be required

Reply:
I updated the `@redhat-cloud-services/types` version to `2.0.0-beta.1`, but with these `any` castings removed I'm still seeing an error at build time: 
![image](https://github.com/user-attachments/assets/89439ecd-c0dd-407a-9693-5bc6286c4ddc)

Reply:
OK. We will can keep it then,